templates/submission/form/section.tpl

```diff
@@ -14,3 +14,43 @@
 {fbvFormSection title="section.section" required=true}
 	{fbvElement type="select" id="sectionId" label=$sectionDescription from=$sectionOptions selected=$sectionId translate=false disabled=$readOnly size=$fbvStyles.size.MEDIUM required=true}
 {/fbvFormSection}
+
+{foreach from=$sectionPolicies key="sectionPolicySectionId" item="content"}
+	{assign var=sectionPolicyHidden value=false}
+	{if $sectionPolicySectionId != $sectionId}
+		{assign var=sectionPolicyHidden value=true}
+	{/if}
+	{include file="submission/form/sectionPolicy.tpl" sectionId=$sectionPolicySectionId content=$content hidden=$sectionPolicyHidden}
+{/foreach}
+
+<script type="text/javascript">
```

Comment:
We're between two Javascript frameworks -- our own homebrew one, which we're gradually working to replace, and a new vue.js-based one. The old framework discouraged Javascript code in template files, and the new one favours vue templates. Maybe @NateWr has a recommendation for this...?

Reply:
Yep, i definitely need some help here on how to implement the dynamic refresh without a javascript block. It is somewhat a quick and dirty solution.

Reply:
Not relevant anymore due to PR provided by @NateWr